app/app_test.go

```diff
@@ -143,3 +146,13 @@ func unmarshalJSONKeys(jsonBytes []byte) ([]string, error) {
 
 	return keys, nil
 }
+
+func removeIbcTmModule(modules []string) []string {
+	var result []string
+	for _, str := range modules {
+		if str != "07-tendermint" {
```

Comment:
We can use `"github.com/cosmos/ibc-go/v7/modules/light-clients/07-tendermint".ModuleName` here